Cargo.toml

```diff
@@ -120,7 +120,7 @@ rand = "=0.8.5"
 regex = "^1.7.0"
 lazy-regex = "3"
 reqwest = { version = "0.11.20", default-features = false, features = ["rustls-tls", "stream", "gzip", "brotli", "socks", "json"] }
-ring = "=0.16.20"
+ring = "0.17.0"
```

Comment:
Could we please bump to `0.17.2` instead?

Reply:
Or `^0.17.0`?

Reply:
Yeah, I think we should do that, but we're still blocked by upstream crate (rustls) before we'll land this PR.

Reply:
@bartlomieju FYI https://github.com/rustls/rustls/releases/tag/v%2F0.21.8

Reply:
Thanks. @littledivy can you finish the update?

Reply:
Updated deno_cache_dir, deno_lockfile to use latest ring.

But we are now blocked on "sct": rustls 0.21.8 depends on sct 0.7.0 which depends on older ring.
So i guess we just wait for yet another rustls which bumps sct :/ https://github.com/rustls/sct.rs/issues/16#issuecomment-1777421251

Reply:
oof my dumb ass forgot about patch semver versioning 😅  Just `cargo update -p sct` worked